### ISSUE TITLE
[DOC] Minor typos

### ISSUE THREAD

**Issue by wiseweb-works** [bunkerity/bunkerweb#290]
**Description**
Some lines are misspelled in Bunkerweb's [docs page](https://docs.bunkerweb.io/1.4/web-ui/#installation) , [quick-start guide](https://docs.bunkerweb.io/1.4/quickstart-guide/#custom-configurations) and github [web-uı.md](https://github.com/bunkerity/bunkerweb/blob/master/docs/web-ui.md) file. (For Docker)

**Proposal (optional)**
1. Need to add `\` sign ın mybunker section , otherwise the code gives an error.
```
docker run -d \
   --name mybunker  <<<<<
   --network bw-services \
   -- ... \
```
2.  Need to add `\` sign after the .../changeme/ section, otherwise the code gives an error.
```
-- ... \
-e ADMIN_PASSWORD=changeme \
-e ABSOLUTE_URI=http(s)://bwadm.example.com/changeme/ <<<<<
bunkerity/bunkerweb-ui:1.4.3 && \
```

3. Need to add `\` sign after the ` MULTISITE`  section, otherwise the code gives an error.
```
-p 80:8080 \
-p 443:8443 \
-v bw-data:/data \
-e MULTISITE=yes  <<<<<
-e "SERVER_NAME=app1.example.com app2.example.com app3.example.com" \
```

4. Not sure, but Web-UI 403 issue (#269 ) seems still unfixed for us after version 1.4.3. If we still need to change `http://myui:7000 \` to `http://127.0.0.1:7000 \`, we also need to change it in the documentation.

**Comment by TheophileDiot** [comment]
Hi, thank you for your issue, we're working on it !

**Comment by TheophileDiot** [comment]
> Not sure, but Web-UI 403 issue (https://github.com/bunkerity/bunkerweb/issues/269 ) seems still unfixed for us after version 1.4.3. If we still need to change http://myui:7000 \ to http://127.0.0.1:7000 \, we also need to change it in the documentation.

@wiseweb-works  This is indeed fixed with the version 1.4.3 because the docker integration still needs to access the web ui through the hostname.

**Comment by fl0ppy-d1sk** [comment]
Let's keep it open until it's merged into master.

I can confirm that `http://myui:7000` is for Docker and `http://127.0.0.1:7000` is for Linux (which is good in the documentation).

**Comment by fl0ppy-d1sk** [comment]
Hello @wiseweb-works,

It should be fixed in the last [v1.4.4 release](https://github.com/bunkerity/bunkerweb/releases/tag/v1.4.4).

**Comment by wiseweb-works** [comment]
@fl0ppy-d1sk Thank you for your all effort
